scripts/build.py

```diff
@@ -46,10 +46,10 @@ def is_windows():
 BUILD_ROOT = os.path.abspath(
     os.path.join(os.path.split(os.path.abspath(__file__))[0], '..'))
 BUILD_CONFIGS = {'debug': 'dbuild', 'release': 'build'}
-CMAKE_VERSION_3_13 = distutils.version.StrictVersion('3.13.0')
+CMAKE_VERSION_3_13 = packaging.version.Version('3.13.0')
```

Comment:
This is only used to compare CMake version at one point.  It's not clear to me why using the full `Version` implementation is superior to a simple integer list comparison:
```
CMAKE_VERSION_3_13 = [3, 13, 0]
```
with corresponding changes to `cmake_version`:
```
cmake_version = [int(x) for x in match.group('version').split('.')]
```